src/Compute/Compute.Test/ScenarioTests/VirtualMachineExtensionTests.cs

```diff
@@ -115,9 +115,16 @@ public void TestAzureDiskEncryptionExtensionSinglePassDisableAndRemove()
 
         [Fact]
```

Comment:
What is AzureRM.Compute.zip?  Please remove this.

Reply:
Sorry, that got added accidentally. Removed this in my last commit.